OasisDEX/oasis-borrow
# DSR rate update banner

# [DSR rate update banner](https://app.shortcut.com/oazo-apps/story/9566/add-dsr-hike-banner-to-homepage-and-all-maker-pages)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added dsr banner to maker positions
- replaced aave banner on the landing page with dsr banner
  
## How to test 🧪
  <Please explain how to test your changes>

- visit maker borrow, multiply, guni position to see dsr banner in overview section
- visit landing page
